app/templates/application.hbs

```diff
@@ -6,9 +6,10 @@
   {{outlet}}
 </main>
 <aside>
-  <p id="detail-list-title">Potential Failures ({{@model.length}})
+  <div id="list-detail-switcher" title="Potential Failures List" aria-label="Potential Failures Menu Switcher" onclick="document.querySelector('aside').classList.toggle('active')"></div>
```

Comment:
- let's make this a `<button>` element so it will be accessible.

Reply:
Oh sorry for that. Fixed.